plugins/inputs/statsd/statsd.go

```diff
@@ -283,9 +285,11 @@ func (s *Statsd) udpListen() error {
 				log.Printf("ERROR READ: %s\n", err.Error())
 				continue
 			}
+			bufCopy := make([]byte, n)
```

Comment:
I could be reading this wrong, but these comments make me think this isn't the correct use-case of a Pool (from https://golang.org/pkg/sync/#Pool):

```
An appropriate use of a Pool is to manage a group of temporary items silently shared among and potentially reused by concurrent independent clients of a package. Pool provides a way to amortize allocation overhead across many clients.

 ...

On the other hand, a free list maintained as part of a short-lived object is not a suitable use for a Pool, since the overhead does not amortize well in that scenario. It is more efficient to have such objects implement their own free list.
```